src/langcheck/metrics/en/query_based_text_quality.py

```diff
@@ -226,3 +226,45 @@ def harmful_activity(
             "High Risk": 1.0,
         },
     )
+
+
+def prompt_leakage(
```

Comment:
nit: I think it's fine either way, but this feels more like a reference-free metric than a query-based metric to me, wdyt?

Reply:
Got it, done in [41327bb](https://github.com/citadel-ai/langcheck/pull/152/commits/41327bbd7c64891c18aece51d9b8dcbdef38d93e).